src/modinv32.h

```diff
@@ -35,4 +35,8 @@ static void secp256k1_modinv32_var(secp256k1_modinv32_signed30 *x, const secp256
 /* Same as secp256k1_modinv32_var, but constant time in x (not in the modulus). */
 static void secp256k1_modinv32(secp256k1_modinv32_signed30 *x, const secp256k1_modinv32_modinfo *modinfo);
 
+/* Compute the Jacobi symbol for (x | modinfo->modulus). Either x must be 0, or x must be coprime with
```

Comment:
maybe this should spell out that the Jacobi symbol is only defined for odd x? (Do we need x > 1?)

Reply:
Okay yes, I don't immediately see why. If this is the case, then it should probably be explained.

(This can be done later as you said below.)

Reply:
Yes, that can be considered resolved. 

> The `secp256k1_modinv32_modinfo` requires an odd modulus already.

I wonder if it's a good idea to add a function `secp256k1_modinv32_modinfo` verify that checks consistency and call it on entry of every function that reads modinfo, similar to how we do this for other data structures. But if yes, that should happen in a separate PR.